tests.py

```diff
@@ -326,6 +360,15 @@ def test_replace(self):
         self.assertEqual(replace(self.b, f=4), self.Beta(1, 2, 4))
         self.assertEqual(self.b, self.Beta(1, 2, 3))  # assert that the original instance remains unchanged
 
+    def test_meta_subclass(self):
+        """Test subclassing of DataClassMeta."""
+        class DataClassMetaSubclass(DataClassMeta):
+            pass
```

Comment:
It might be good to add some custom behaviour here to confirm that it's added to the instantiated class. I don't know if you have any ideas.

Reply:
Good idea, I added it.